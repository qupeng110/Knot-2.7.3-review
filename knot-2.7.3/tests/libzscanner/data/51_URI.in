$ORIGIN	.
$TTL	1

; OK
@	URI	0	0	a		; The simplest variant
@	URI	65535	65535	ftp://a		; Maximal priority and weight
@	URI	0       0	"ftp://a"	; Quoted target
@	URI	0       0	012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789 ; Target longer than 255
@	TYPE256	\# 5 0000000061			; TYPE + Hexadecimal rdata
@	TYPE256	0	0	a		; TYPE
@	uri	0	0	a		; Type in lower-case

; OK extensions
@	URI	0	0	""		; Empty target

; KO
@	URI
@	URI					; Empty rdata
@	URI	\# 0				; Hex empty rdata
@	URI	65536	0	a		; Priority overflow
@	URI	0	65536	a		; Weight overflow
@	URI	0	0	a	a	; Unexpected item
