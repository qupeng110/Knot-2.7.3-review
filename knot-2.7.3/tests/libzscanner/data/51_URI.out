OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=0000000061
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=FFFFFFFF6674703A2F2F61
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=000000006674703A2F2F61
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=00000000303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=0000000061
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=0000000061
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=0000000061
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0100
RDATA=00000000
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_BAD_REST
------
